src/main/scala/org/ergoplatform/nodeView/state/SnapshotsInfo.scala

```diff
@@ -0,0 +1,36 @@
+package org.ergoplatform.nodeView.state
+
+import org.ergoplatform.ErgoLikeContext.Height
+import org.ergoplatform.nodeView.state.UtxoState.ManifestId
+
+/**
+  * Container for UTXO set snapshots the node holds
+  * @param availableManifests - available UTXO set snapshot manifests and corresponding heights
+  */
```

Comment:
What is the key in the map? It doesn't have ScalaDoc

Reply:
height, it is covered in the scaladoc